src/pocketmine/resources/pocketmine.yml

```diff
@@ -121,6 +121,50 @@ chunk-ticking:
  tick-radius: 3
  light-updates: false
  clear-tick-list: true
+ 
+block-ticking:
+ #Configure which blocks should tick and which not
+ #Useful on slow PC's and to disable ice melting, fire burning down things etc.
+ #Grass
```

Comment:
Because configs are meant to be easy for users that set up a server, they may don't know whan block they mean